<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OOStandartMusteriAramaFormu</class>
 <widget class="QDialog" name="OOStandartMusteriAramaFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ada Göre Arama</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblArananDeger">
             <property name="text">
              <string>Aranan Değer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leArananMusteriAdi"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grpAdaGoreAramaSecenekleri">
         <property name="title">
          <string>Arama Seçenekleri</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbIleBaslayan">
            <property name="text">
             <string>İle Başlayan</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rblIleBiten">
            <property name="text">
             <string>İle Biten</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbIceren">
            <property name="text">
             <string>İçeren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbYazarkenAra">
            <property name="text">
             <string>Yazarken Ara</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAra">
           <property name="text">
            <string>Ara</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>TC Numarasına Göre Arama</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Adı Soyadı</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TC Kimlik No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Müşteri Numarası</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Müşteri Bilgileri Düzelt</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Müşteri Sil</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Kapat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>OOStandartMusteriAramaFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
